extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -430,23 +432,23 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		// the top we'll expand nodes until we find the parent of the Notebook we're looking for
 		// get the children of root node and expand the nodes to the notebook level.
 		await this.getChildren(parentBook.rootNode);
-		// The path to the parent of the Notebook we're looking for (this is the node we're looking to expand)
-		const parentPath = notebookPath.substring(0, notebookPath.lastIndexOf(path.posix.sep));
+		// The path to the Notebook we're looking for (these are the nodes we're looking to expand)
+		const notebookFolders = notebookPath.split('/');
 		// Find number of directories between the Notebook path and the root of the book it's contained in
 		// so we know how many parent nodes to expand
 		let depthOfNotebookInBook: number = path.relative(notebookPath, parentBook.bookPath).split(path.sep).length;
 		// Walk the tree, expanding parent nodes as needed to load the child nodes until
 		// we find the one for our Notebook
-		while (depthOfNotebookInBook > 0) {
+		while (depthOfNotebookInBook > -1) {
```

Comment:
So while we're here I think it would be good to clean up the logic and make it clearer what's happening. Having the loop depend on `depthOfNotebookInBook` is pretty confusing - since that's a value I'd expect never to change. 

I'd suggest adding a new var `currentDepth` that starts at 0 and goes until it === `depthOfNotebookInBook`, with the loop incrementing `currentDepth` each time. 

Then you could just do `let parentBookPath: string = notebookFolders.slice(0, currentDepth).join('/');`